dbms/src/Flash/Mpp/AsyncRequestHandler.h

```diff
@@ -280,22 +283,22 @@ class AsyncRequestHandler : public AsyncRequestHandlerBase
         rpc_context->makeAsyncReader(request, reader, cq, thisAsUnaryCallback());
     }
 
-    bool retryOrDone(String done_msg)
+    void retryOrDone(String && done_msg, const String & log_msg)
     {
         if (retriable())
         {
+            LOG_WARNING(log, log_msg);
```

Comment:
Move to L287 and don't need two `LOG_WARNING(log, log_msg)`?

Reply:
> Move to L287 and don't need two `LOG_WARNING(log, log_msg)`?

okk